Makefile

```diff
@@ -263,10 +263,9 @@ manifests: ## Generate K8s manifests e.g. CRD, RBAC etc.
 	$(eval TMPDIR := $(shell mktemp -d -t cilium.tmpXXXXXXXX))
 	$(QUIET)$(GO) run sigs.k8s.io/controller-tools/cmd/controller-gen $(CRD_OPTIONS) paths=$(CRD_PATHS) output:crd:artifacts:config="$(TMPDIR)"
 	$(QUIET)$(GO) run ./tools/crdcheck "$(TMPDIR)"
-
-	# Clean up old CRD state and start with a blank state.
+	@
```

Comment:
Typically we'd use `$(QUIET)` instead, so that this respects `V=1`.

Also, any reason to drop the comment here? The context is useful to understand why we're deleting the CRDs.

Reply:
oh yeah, so it turns out the reason why the `make manifest` was failing was because of the comment in here! I tried `@ # Clean...` and it also failed :exploding_head:

Reply:
:raised_eyebrow: Does it make a difference whether it is `@# comment` vs. `@ # comment`? We seem to use `@# comment` in various other targets already. Or do you also experience the issue for those targets, such as API generation?

Reply:
I'm also wondering, are the new use of `@` introduced here necessary to resolve your issue, or just part of the debugging process? I'm not sure why the specific two lines here that you've added `@` to need this, and why the others wouldn't. Shouldn't each of the for loops be handled similarly for echoing the command to stdout?

Reply:
The other trick we always have up our sleeve is just to move some of this loop logic into a separate bash script and invoke the script.

Reply:
> 🤨 Does it make a difference whether it is `@# comment` vs. `@ # comment`? We seem to use `@# comment` in various other targets already. Or do you also experience the issue for those targets, such as API generation?

No, both fail. The only one that doesn't fail is if I remove the comment altogether. I just added the `@` to be consistent with the other targets.



> The other trick we always have up our sleeve is just to move some of this loop logic into a separate bash script and invoke the script.

Yeah I think I'll end up doing that.

Reply:
Done